elasticsearch/run.sh

```diff
@@ -162,13 +162,11 @@ seed_searchguard(){
         -tspass tspass \
         -nhnv \
         -icl ${DEBUG:+-dg}
-    
-    if [ $? -eq 0 ]; then
-      info "Seeded the searchguard ACL index"
-    else
-      error "Error seeding the searchguard ACL index"
-      exit 1
-    fi
+    do
+        warn "Error seeding the searchguard ACL index... retrying in 10 seconds"
+        sleep 10
+    done
+    info "Seeded the searchguard ACL index"
```

Comment:
The timeout today is killing us.  If the sgadmin script does not eventually succeed, then we are left in a lurch.  Nothing can move forward.  This way, one can correct the environment if possible to enable the sgadmin script to succeed (like fixing bad indices, or other pods causing a problem, etc.).